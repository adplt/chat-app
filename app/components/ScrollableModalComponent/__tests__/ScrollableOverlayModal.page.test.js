import React from 'react';
import renderer from 'react-test-renderer';
import ScrollableOverlayModal from '../ScrollableOverlayModal.component';

describe('ScrollableOverlayModal component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ScrollableOverlayModal />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
