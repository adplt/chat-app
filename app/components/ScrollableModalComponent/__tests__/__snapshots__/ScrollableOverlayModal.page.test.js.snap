// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollableOverlayModal component renders correctly 1`] = `null`;
